### PR TITLE
Adds a new lowpop (Crash) map, Desparity.

### PR DESCRIPTION
**Pull request by chizzy376**
## About The Pull Request
Good morning Marines! We have received a distress signal from a freshly established colony similar to LV624! Unfortunate thing is, you are the source of the signal, good luck surviving in this harsh environment!
map

**The map uses weedable jungle grass, so you won't get the LV624 jungle experience**

--
## Why It's Good For The Game
More map variety good, need to get this out here so people can properly critique the map after playing it. Much better to get player feedback rather than speculation.
## Changelog
:cl:
add: Added a new map, Desparity. Lowpop exclusive (30 pop or less).
/:cl:

### CODE REVIEW

**Comment by Losenis** [comment]
>All that green

That better be weedable

**Comment by chizzy376** [comment]
> > All that green
> 
> That better be weedable

It is

File: code/game/turfs/floor_ground.dm [hunk 1]
```diff
@@ -109,6 +110,8 @@
 	mediumxenofootstep = FOOTSTEP_GRASS
 	smoothing_groups = list(SMOOTH_GROUP_JUNGLE_FLOOR)
 
+/turf/open/ground/grass/weedable
+	name = "soft grass"
```
Comment (Lumipharon) [inline code/game/turfs/floor_ground.dm:114]:
```suggestion
	name = "soft grass"
	
```

**Review by Lumipharon — APPROVED** [review]

File: code/game/turfs/floor_ground.dm (hunk 1 above)
Comment (chizzy376) [reply code/game/turfs/floor_ground.dm:114]: Done

**Comment by UntoldTactics** [comment]
Decent map really, though I think it needs to be cut down in regards to large open areas. Maps designed for crash should not have that much area inherently advantageous to marines on the get-go, at least, not so close to disks. Only takes a single flamer to reduce any maze that could be made in crash into nothing.

**Comment by chizzy376** [comment]
> Decent map really, though I think it needs to be cut down in regards to large open areas. Maps designed for crash should not have that much area inherently advantageous to marines on the get-go, at least, not so close to disks. Only takes a single flamer to reduce any maze that could be made in crash into nothing.

Which areas specifically? I'll assume the spot south of engi, east of green? I could add a building there in place of the wide sand path.

**Comment by UntoldTactics** [comment]
> Which areas specifically? I'll assume the spot south of engi, east of green? I could add a building there in place of the wide sand path.

Pretty much. Crash is a pretty unmaintained mode, there's a specific 'goldilocks zone' where xenos have lower pop than marine numbers, which results in open areas essentially becoming death zones. I haven't played the map enough to highlight spots, though just wanted to bring up awareness that open areas are a significant advantage towards marines and disadvantage to xenos.

**Comment by chizzy376** [comment]
> > Which areas specifically? I'll assume the spot south of engi, east of green? I could add a building there in place of the wide sand path.
> 
> Pretty much. Crash is a pretty unmaintained mode, there's a specific 'goldilocks zone' where xenos have lower pop than marine numbers, which results in open areas essentially becoming death zones. I haven't played the map enough to highlight spots, though just wanted to bring up awareness that open areas are a significant advantage towards marines and disadvantage to xenos.

Updated the map, updated the pictures as well

**Comment by chizzy376** [comment]
Someone fix this shit yourself, I don't get paid to slave over some moron's "CI Suite / Compile Maps (pull_request) Failing after 15s", who the fuck made this shit, I didn't change anything in this pr it was working yesterday, fuck off

**Pull request merged**